Tests/NIOTests/CircularBufferTests.swift

```diff
@@ -207,4 +207,70 @@ class CircularBufferTests: XCTestCase {
             XCTAssertEqual(idx + 25, element)
         }
     }
+
+    func testCount() {
+        var ring = CircularBuffer<Int>(initialRingCapacity: 4)
+        ring.append(1)
+        XCTAssertEqual(1, ring.count)
+        ring.append(2)
+        XCTAssertEqual(2, ring.count)
+        XCTAssertEqual(1, ring.removeFirst())
+        ring.append(3)
+        XCTAssertEqual(2, ring.count)
+        XCTAssertEqual(2, ring.removeFirst())
+        ring.append(4)
+
+
+        XCTAssertEqual(2, ring.count)
+        XCTAssertEqual(3, ring.removeFirst())
+        ring.append(5)
+
+        XCTAssertEqual(3, ring.startIdx)
+        XCTAssertEqual(1, ring.endIdx)
+        XCTAssertEqual(2, ring.count)
+        XCTAssertEqual(4, ring.removeFirst())
+        XCTAssertEqual(5, ring.removeFirst())
+        XCTAssertEqual(0, ring.count)
+        XCTAssertTrue(ring.isEmpty)
+    }
 }
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Um?

Reply:
more empty lines ftw... seriously let me cleanup while adding more tests

Reply:
I wonder if I felt asleep while holding the enter key 🤷‍♂️